### PR TITLE
update stripbuildinfo.patch

### PR DESCRIPTION
**Pull request by fanquake**

### CODE REVIEW

File: patches/stripbuildinfo.patch [hunk 1]
```diff
@@ -113,25 +86,24 @@ index 0c6f0ef0a..97c6ffe8b 100644
...
-diff --git a/src/validation.cpp b/src/validation.cpp
-index 9f5c59e52..bda70c649 100644
---- a/src/validation.cpp
-+++ b/src/validation.cpp
-@@ -52,9 +52,6 @@
- #include <boost/algorithm/string/replace.hpp>
- #include <boost/thread.hpp>
-
--#if defined(NDEBUG)
--# error "Bitcoin cannot be compiled without assertions."
```
Comment (MarcoFalke) [inline patches/stripbuildinfo.patch:133]: Does this no longer need to be removed?

**Comment by laanwj** [comment]
Ping @fanquake re: @MarcoFalke's comment

**Comment by fanquake** [comment]
Thanks @MarcoFalke and @laanwj. The previous patch was not correct. Have updated and performed some new builds.

**Comment by MarcoFalke** [comment]
review ACK e692ade94427cf80894ab9c75399c220230bb501 

didn't test

**Pull request merged**